tests/10-deploy-test

```diff
@@ -63,5 +63,22 @@ class TestDeployment(unittest.TestCase):
             message = 'Could not delete the busybox container.'
             amulet.raise_status(amulet.FAIL, msg=message)
 
+    def test_latest_config_option(self):
```

Comment:
The tests are run in alphanumeric order and some tests may need to run in a specific sequence and should not impose rules on them.

I think is is a very good best practice if ordering is not required.